controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/archive/ArchiveService.java

```diff
@@ -4,20 +4,18 @@
 import com.yahoo.config.provision.TenantName;
 import com.yahoo.config.provision.zone.ZoneId;
 
-import java.net.URI;
-import java.util.Optional;
+import java.util.Map;
+import java.util.Set;
 
 /**
  * Service that manages archive storage URIs for tenant nodes.
  *
  * @author freva
+ * @author andreer
  */
 public interface ArchiveService {
 
-    Optional<URI> archiveUriFor(ZoneId zoneId, TenantName tenant);
-
-    // TODO: Method to configure archive permissions/access for a tenant
-
-    // TODO: Method to revoke permission/access for a tenant
+    ArchiveBucket createArchiveBucketFor(ZoneId zoneId, Set<TenantName> tenantName);
```

Comment:
The tenant names doesn't really belong in this API, with the `.withTenant()` method (see other comment) the user should just call `createArchiveBucketFor(zoneId).withTenant(tenantName)`